.github/workflows/new-cfe-cherry-pick.yml

```diff
@@ -235,6 +235,16 @@ jobs:
 
                       ${{ steps.cherry-pick-pr.outputs.cherry-pick-pr }}
 
+            - name: Apply label "cherry pick failed" using the newly created PR url
```

Comment:
I was thinking if we can add a check for this in the build zip workflow as well, wdyt?